contrib/babelfishpg_tsql/src/iterative_exec.c

```diff
@@ -1253,6 +1255,7 @@ dispatch_stmt_handle_error(PLtsql_execstate *estate,
 		int			last_error;
 		bool		error_mapped;
 
+		HOLD_INTERRUPTS();
```

Comment:
it would be better to put hold and resume at individual code blocks rather than whole CATCH block

Reply:
Yes, good point